sdk/nodejs/dynamodb/dynamodbMixins.ts

```diff
@@ -28,10 +28,38 @@ export interface TableEventSubscriptionArgs {
      */
     readonly batchSize?: number;
 
+    /**
+     * The maximum amount of time to gather records before invoking the function, in seconds. Records will continue to buffer
+     * until either maximum_batching_window_in_seconds expires or batch_size has been met. Defaults to as soon as records
+     * are available in the stream. If the batch it reads from the stream only has one record in it, Lambda only sends one record to the function.
+     */
+    readonly maximumBatchingWindowInSeconds?: number;
+
     /**
      * The position in the stream where AWS Lambda should start reading. Must be one of either `TRIM_HORIZON` or `LATEST`.
      */
     readonly startingPosition: "TRIM_HORIZON" | "LATEST";
+
+    /**
+     * The number of batches to process from each shard concurrently. Minimum and default of `1`, maximum of `10`
+     */
+    readonly parallelizationFactor?: number;
+
+    /**
+     * The maximum number of times to retry when the function returns an error. Minimum of `0`, maximum and default of `10000`.
+     */
+    readonly maximumRetryAttempts?: number;
+
+    /**
+     * The maximum age of a record that Lambda sends to a function for processing. Minimum of `60`, maximum and default of `604800`
+     */
+    readonly maximumRecordAgeInSeconds?: number;
+
+    /**
+     * If the function returns an error, split the batch in two and retry. Defaults to `false`.
+     */
+    readonly bisectBatchOnFunctionError?: boolean;
```

Comment:
Do you mind sorting them alphabetically to be able to easily compare to the source list? Thanks!

Reply:
done!